rest/model/project.go

```diff
@@ -531,12 +534,12 @@ func (p *APIProjectRef) ToService() (*model.ProjectRef, error) {
 		projectRef.PatchTriggerAliases = patchTriggers
 	}
 
-	if p.ContainerSizes != nil {
-		containerSizes := map[string]model.ContainerResources{}
-		for name, size := range p.ContainerSizes {
-			containerSizes[name] = size.ToService()
+	if p.ContainerSizeDefinitions != nil {
```

Comment:
Since this field is `omitempty` in both the DB and the API models, it cannot be an empty slice, so this could be simplified down to remove the if statement:
```go
for _, size := range p.ContainerSizeDefinitions {
    projectRef.ContainerSizeDefinitions = append(projectRef.ContainerSizeDefinitions, size.ToService())
}
```

Reply:
Good catch, done